content/_partials/toptoolbar.html.haml

```diff
@@ -43,23 +43,17 @@
             - link = expand_link("solutions/#{key}")
             %a.dropdown-item.feature{:href => link, :class => active_link?(key.to_s)}
               = site.solutions[key].usecase
-      %li.nav-item.dropdown
+      %li.nav-item.dropdown{:class => ('active' if (page.section == 'participate'))}
         %a.nav-link.dropdown-toggle{'data-toggle' => 'dropdown',
             :href=> "#", :role=>"button", 'aria-haspopup'=>"true", 'aria-expanded'=>"false"}
           Participate
         %div.dropdown-menu
+          %a.dropdown-item{:href => expand_link("participate"), :title => "Participate and contribute"}
+            Overview
           %a.dropdown-item{:href => expand_link("content/mailing-lists"), :title => "Browse Jenkins mailing list archives and/or subscribe to lists"}
             Mailing Lists
           %a.dropdown-item{:href => expand_link("content/chat"), :title => "Chat with the rest of the Jenkins community on IRC"}
             Chat
-          %a.dropdown-item{:href => "https://twitter.com/jenkinsci", :title => "Check out @jenkinsci on Twitter"}
```

Comment:
I don't really see the "participation" in Twitter ("tweet at us how much you enjoy Jenkins" or what?), and it's referenced from the footer anyway.